BRAD/scraper.py

```
@@ -134,35 +148,158 @@ def webScraping(state):
         
-        state['process']['steps'].append(process)
-        state['output'] = "Articles were successfully downloaded."
-    else:
-        state['output'] = "No articles were searched."        
+    state['process']['steps'].append(process)
+    return state
+
+
+def webScrapingStageTwo(state):
+    query    = state['prompt']
+    print(f"webScrapingStageTwo")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)